src/shapes/tessellation_twist/TessellationTwist.js

```diff
@@ -0,0 +1,266 @@
+import Victor from 'victor'
+import { Graph, vec2, mix } from '../../common/Graph'
+import eulerianTrail from '../../common/eulerianTrail'
+import { difference } from '../../common/util'
+import Shape, { shapeOptions } from '../Shape'
+
+const vec_triangle = [
```

Comment:
These vertices seem a lot like the math I worked out for the releaux triangle. Although that was the center of the arc circles, so it is rotated a bit. IDK if higher number of sides is interesting to you, but if this is the limit, we could probably generalize this for almost any number of sides (greater than three).

Reply:
That would be cool. I don't think I could work out the math myself though :)